react/utils/pixel.ts

```diff
@@ -27,27 +31,29 @@ export function handleItemClick(
   page: string,
   type: string
 ) {
-  return (term: string, position: number) => {
+  return (term: string, position: number, products: Item[]) => {
```

Comment:
the items are not products, but terms.. you really need to have the info about all the terms suggested when clicking in a specific term? if so, could you rename this prop accordingly?